editoast/editoast_models/src/db_connection_pool.rs

```diff
@@ -262,41 +270,39 @@ impl DbConnectionPoolV2 {
     #[allow(unused)] // TEMPORARY
     pub fn iter_conn(
         &self,
-    ) -> impl Iterator<Item = impl Future<Output = Result<DbConnectionV2, EditoastModelsError>> + '_>
+    ) -> impl Iterator<Item = impl Future<Output = Result<DbConnectionV2, DatabasePoolError>> + '_>
     {
         std::iter::repeat(self).map(|p| p.get())
```

Comment:
I know it's not part of the PR... but you can write this in one function call I believe.

```suggestion
        std::iter::repeat_with(|| self.get())
```

Reply:
It's not a big change and it's close to a diff of the PR so imo it's a relevant addition :)